src/constants/common.js

```diff
@@ -171,6 +173,7 @@ module.exports = {
 	sessionCompleteEndpoint: '/mentoring/v1/sessions/completed/',
 	INACTIVE_STATUS: 'INACTIVE',
 	ACTIVE_STATUS: 'ACTIVE',
+	SEARCH: 'a',
```

Comment:
What does this do ? If not needed remove this please

Reply:
I fetching Search string from common . i need this @nevil-mathew